dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/BaseControls/CSSCST.mo

```diff
@@ -0,0 +1,117 @@
+within Dynawo.Electrical.StaticVarCompensators.BaseControls;
+
+/*
+* Copyright (c) 2023, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+model CSSCST "Static var compensator control model with voltage override as susceptance command for switched shunts"
+  import Modelica;
+  import Dynawo;
+  import Dynawo.Types;
+
+  parameter Real BMax "Maximum capacitive output of the SVarC in S";
+  parameter Real BMin "Maximum inductive output of the SVarC in S";
+  parameter Types.PerUnit K "Control gain constant";
+  parameter Types.ApparentPowerModule SBase = 1 "Base apparent power in MVA";
```

Comment:
It was there in the original model. SBase is not given in any report. In the original model as well as in this one, we have BRef0Pu = -BVar0Pu * 100 / K. Since URef0Pu = U0Pu and UOtherPu = 0, it would be consistent to have SBase = 100 MVA = SnRef.

Reply:
ok

Reply:
BRefPu is therefore in (base UNom, SBase = 1), which is weird.